celements-model/src/main/java/com/celements/model/context/ModelContext.java

```diff
@@ -130,6 +134,12 @@
   @NotNull
   Optional<XWikiResponse> getResponse();
 
+  /**
+   * @return the current language or null
+   */
+  @Nullable
+  String getLanguage();
```

Comment:
please use `java.util.Optional` instead uf null pointer

Reply:
Had to use java.util.Optional since many methods use import com.google.common.base.Optional and I didn't want to refactor several projects to change it